frontend/src/App.test.tsx

```diff
@@ -1,9 +1,8 @@
-import { render, screen } from "@testing-library/react";
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable react/react-in-jsx-scope */
+import { render } from "@testing-library/react";
 import App from "./App";
-
-test("renders learn react link", () => {
-  // eslint-disable-next-line react/react-in-jsx-scope
+//@ts-ignore
```

Comment:
Why do we need the ts-ignore's? We were running type checks before so I'm confused as to why something would stop working when updating GitHub Actions workflows.